File: 프로그래머스/0/120829. 각도기/각도기.py
def solution(angle):
    #if angle > 0 and angle < 90:
    if 0 < angle < 90:      # 예각
        return 1
    elif angle == 90:       # 직각
        return 2
    #elif angle > 90 and angle < 180:
    elif 90 < angle < 180:  # 둔각
        return 3
    elif angle == 180:      # 평각
        return 4

# ✅ 예제 테스트 실행
print(solution(70))   # 결과: 1
print(solution(91))   # 결과: 3
print(solution(180))  # 결과: 4

File: 프로그래머스/0/120829. 각도기/각도기.java
class Solution {
    public int solution(int angle) {
        int answer = 0;

        if (angle < 90) {
            answer = 1; // 예각
        } else if (angle == 90) {
            answer = 2; // 직각
        } else if (angle < 180) {
            answer = 3; // 둔각
        } else {
            answer = 4; // 평각
        }

        return answer;
    }

    public static void main(String[] args) {
        Solution s = new Solution();

        // 테스트 예제 실행
        System.out.println(s.solution(70));  // 1
        System.out.println(s.solution(91));  // 3
        System.out.println(s.solution(180)); // 4
    }
}